java/src/jmri/jmrix/loconet/LocoNetConsistManager.java

```diff
@@ -54,13 +55,13 @@ public boolean csConsistNeedsSeperateAddress() {
      * consistTable/consistList
      */
     @Override
-    public Consist addConsist(DccLocoAddress address) {
+    public Consist addConsist(LocoAddress address) {
         if (consistTable.containsKey(address)) // no duplicates allowed.
         {
             return consistTable.get(address);
         }
         LocoNetConsist consist;
-        consist = new LocoNetConsist(address, memo);
+        consist = new LocoNetConsist((DccLocoAddress)address, memo);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.